decentraland/unity-renderer
# implementation of join channel error modal for unknown reason

## What does this PR change?

Adds a modal that is shown whenever the join channel operation fails for unknown reasons.

## How to test the changes?

1. The easiest way is to modify `ChatController.JoinOrCreateChannel` method forcing a `JoinChannelError(Errorcode.Unknown)`
2. Go to the channel browser
3. Click the join button on any channel
4. A Modal indicating the error should be shown

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
